<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaAcquisto</class>
 <widget class="QWidget" name="VistaAcquisto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data di acquisto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_importo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Importo</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_punti">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Punti</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fumetti acquistati:</string>
   </property>
  </widget>
  <widget class="QListView" name="lista_fumetti">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modifica">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifica</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_elimina">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>450</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
